CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ add_library(EAThread ${EATHREAD_SOURCES})
 
 if(EATHREAD_BUILD_TESTS)
     add_subdirectory(test)
+else()
+    add_subdirectory(test/packages/EABase)
```

Comment:
You shouldn't need to include this.  It EABase should be using its own headers rather than a clone in the packages directory.